game/graphics/pipelines/opengl.cpp

```diff
@@ -10,6 +10,8 @@
 #include "third-party/imgui/imgui.h"
 #include "third-party/imgui/imgui_impl_glfw.h"
 #include "third-party/imgui/imgui_impl_opengl3.h"
+#define STB_IMAGE_IMPLEMENTATION
+#include "third-party/stb_image.h"
```

Comment:
thanks for adding this library, this is a useful one.  I think we're going to want it in more places (for loading/saving texture files, screenshots, etc).  Can you move the implementation to `common`?  I think you would just remove the `#define STB_IMAGE_IMPLEMENTATION` from here, and then make a new file in `common` that just contains:
```
#include "third-party/stb_image.h"
#define STB_IMAGE_IMPLEMENTATION
```